exercises/python-1-a/README.md

```diff
@@ -15,6 +15,14 @@ For example:
 b'\x01a\x02b\x03c\x04d\x06a'
 ```
 
+## Debugging
```

Comment:
The debugging instructions were missing for Python (I've checked that it does indeed work).

Reply:
Great. Thanks :)